Amazon.Ion.ObjectMapper/IonSerializer.cs

```diff
@@ -164,94 +164,67 @@ public void Serialize<T>(IIonWriter writer, T item)
             if (item == null)
             {
                 new IonNullSerializer().Serialize(writer, (object)null);
-                return;
             }
-
-            if (item is bool)
+            else if (item is bool)

```

Comment:
Why did you change this style?

Reply:
The style change is an artifact of a previous version when we were tracking `currentDepth`. Rather than decrementing `currentDepth` in each if block, it got decremented just once at the end. 
While it's a moot point now since we dropped `currentDepth` (and also because another PR makes some significant changes to this section), I will revert this back to the old style.

Reply:
Reverted back to old style.